### ISSUE TITLE
The attention mask and the pad token id were not set.

### ISSUE THREAD

**Issue by wayne810** [starmpcc/Asclepius#7]
Hi Junu Kim,

A promblem happen when I run how_to_use.py. I list the info blow:

Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████| 7/7 [00:05<00:00,  1.32it/s]
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

Can you give me some advice?

Best,
Wayne

**Comment by wayne810** [comment]
By the way, the model is Asclepius-Llama3-8B downloaded from huggingface

**Comment by wayne810** [comment]
I find it dosen't matter, see https://github.com/QwenLM/Qwen2.5/issues/133
